std/assembly/rt/index.d.ts

```diff
@@ -10,3 +10,4 @@ declare function __visit(ref: usize, cookie: i32): void;
 declare function __visit_globals(cookie: u32): void;
 declare function __visit_members(ref: usize, cookie: u32): void;
 declare function __allocArray(length: i32, alignLog2: usize, id: u32, data?: usize): usize;
+declare const ASC_RTRACE: boolean;
```

Comment:
this declaration better include [here](https://github.com/AssemblyScript/assemblyscript/blob/master/std/assembly/index.d.ts#L60) I guess

Reply:
note that this is actually not present at all without `--use ASC_RTRACE=1`, and meant to be used as `isDefined(ASC_RTRACE)`.

Reply:
Which is my I thought it best to put here since this is the folder where it is referenced and so IDE's will use it.  And it seemed like an internal definition so I didn't want to put it in the higher level file.